app/controllers/claims_controller.rb

```diff
@@ -80,6 +76,6 @@ def page_sequence
   end
 
   def claim_slug_sequence
-    current_claim.eligibility.class.parent::SlugSequence.new(current_claim)
+    current_claim.policy::SlugSequence.new(current_claim)
```

Comment:
Technically it could, but I think there's little in it in terms of the benefit of doing so. Doing so would mean that the `Claim` model now has knowledge and responsibility over slug sequences, taking it a step closer to becoming a god class. Right now it's a data model with knowledge about its own validity and eligibility, and the policy it belongs to. I don't think we gain anything from coupling it to sequence code, especially as this is the only place in the application we call this line, so we don't DRY anything up either.